README.md

```diff
@@ -39,9 +39,9 @@ src="https://github.com/common-workflow-language/cwl-website/raw/main/openstand-
 The Common Workflow Language (CWL) is an open standard for describing analysis
 workflows and tools in a way that makes them portable and scalable across a
 variety of software and hardware environments, from workstations to cluster,
-cloud, and high performance computing (HPC) environments.  CWL is designed to
-meet the needs of data-intensive science, such as Bioinformatics, Medical
-Imaging, Astronomy, High Energy Physics, and Machine Learning.
+cloud, and high performance computing (HPC) environments.CWL is designed to
```

Comment:
You've introduced a typo, please fix.

Reply:
Ok

Reply:
I mean you removed the space between "environments.CWL"

Reply:
Done